projects/addon-commerce/components/money/pipes/integer-part.pipe.ts

```diff
@@ -12,7 +12,7 @@ export class TuiIntegerPartPipe implements PipeTransform {
     ) {}
 
     transform(value: number): string {
-        return tuiFormatNumber(Math.floor(Math.abs(Number(value))), {
+        return tuiFormatNumber(Math.trunc(Number(value)), {
```

Comment:
Please, could you use another pull request to have semantic changes in the codebase?